source/scripting/Ped.hpp

```diff
@@ -587,8 +587,8 @@ namespace GTA
 		void RandomizeOutfit();
 		void SetDefaultClothes();
 		void LeaveGroup();
-		void Clone();
-		void Clone(float heading);
```

Comment:
It would work for text scripts. But not for compiled ones, because the IL references the full method including return type.